x-pack/plugins/cloud_security_posture/public/components/fleet_extensions/aws_credentials_form.tsx

```diff
@@ -225,54 +242,158 @@ const getInputVarsFields = (
 const getAwsCredentialsType = (input: Props['input']): AwsCredentialsType | undefined =>
   input.streams[0].vars?.['aws.credentials.type'].value;
 
-export const AwsCredentialsForm = ({ input, newPolicy, updatePolicy }: Props) => {
+const CloudFormationSetup = ({ integrationLink }: { integrationLink: string }) => {
+  return (
+    <>
+      <EuiText color="subdued" size="s">
+        <ol
+          css={css`
+            list-style: auto;
+          `}
+        >
+          <li>
+            <FormattedMessage
+              id="xpack.csp.awsIntegration.cloudFormationSetupStep.login"
+              defaultMessage="Log in as an admin to the AWS Account you want to onboard"
+            />
+          </li>
+          <li>
+            <FormattedMessage
+              id="xpack.csp.awsIntegration.cloudFormationSetupStep.save"
+              defaultMessage="Click the Save and continue button on the bottom right of this page"
+            />
+          </li>
+          <li>
+            <FormattedMessage
+              id="xpack.csp.awsIntegration.cloudFormationSetupStep.launch"
+              defaultMessage="On the subsequent pop-up modal, click the Launch CloudFormation button."
+            />
+          </li>
+        </ol>
+      </EuiText>
+      <EuiSpacer size="l" />
+    </>
+  );
+};
+
+const ReadDocumentation = ({ integrationLink }: { integrationLink: string }) => {
```

Comment:
this naming is a bit misleading, it's not an integration link it's a documentation link. 
also, it would probably be more appropriate if we can actually link to some cloud formation guide instead of general getting started with cspm, if we have that available of course.